pkg/api/register.go

```diff
@@ -70,6 +69,7 @@ func init() {
 	)
 
 	// Register Unversioned types
+	Scheme.AddKnownTypes("", &unversioned.ListOptions{})
```

Comment:
Probably it's ok. We register the ListOptions here because we need to generate the conversion function between unversioned.ListOptions and v1.ListOptions. Because we don't have ListOptions in other groups, so we don't need to register unversioend.ListOptions in other groups.